tsdb/chunks/chunks.go

```diff
@@ -252,34 +257,58 @@ func MergeOverlappingChunks(chks []Meta) ([]Meta, error) {
 			last++
```

Comment:
We don't need this variable we can just use `len(newChks)-1` right?

Reply:
Yes. I was trying to adhere to the existing code but Okay! :)